File: RQ1/GitHub Copilot Chat/C/Type-1 Case.c
// Source Code
/*============================================================*/
/* GRAMMAR RULES */
/*============================================================*/
static void field (LexState *ls, expdesc *v){
  /* field -> ['.' | ':'] NAME */
  FuncState *fs = ls->fs;
  expdesc key;
  luaK_exp2anyreg(fs, v);
  luaX_next(ls);  /* skip the dot or colon */
  checkname(ls, &key);
  luaK_indexed(fs, v, &key);
}

// Synthesized Code
static void field(LexState *ls, expdesc *v) {
    FuncState *fs = ls->fs;
    expdesc key;
    luaK_exp2anyreg(fs, v);
    luaX_next(ls);
    checkname(ls, &key);
    luaK_indexed(fs, v, &key);
}

File: RQ1/GitHub Copilot Chat/Python/Type-1 Case.py
# Source Code
def p_paramdefs(p):
    """paramdefs : '(' paramnames ')'
                 | '(' ')'"""
    if len(p) == 4:
        p[0] = p[2]
    else:
        p[0] = []

# Synthesized Code
def p_paramdefs(p):
    if len(p) == 4:
        p[0] = p[2]
    else:
        p[0] = []